snuba/settings/__init__.py

```diff
@@ -78,16 +83,29 @@
 CLICKHOUSE_TRACE_USER = os.environ.get("CLICKHOUSE_TRACE_USER", "default")
 CLICKHOUSE_TRACE_PASSWORD = os.environ.get("CLICKHOUSE_TRACE_PASS", "")
 
-# Redis Options
 USE_REDIS_CLUSTER = os.environ.get("USE_REDIS_CLUSTER", "0") != "0"
```

Comment:
I'm not a fan of this option but for now I kept it so that you can run local tests as well as tests in CI with SNUBA_SETTINGS=test, though I think the entirety of redis connection settings should live in the `REDIS_CLUSTER_MAP`

Reply:
How hard is it to have CI/local testing use the same Redis connection?

Reply:
I didn't really investigate but it means either using single-node configured Redis in CI or using cluster-configured Redis locally (the two connection types are not compatible)